docs/admin/deployment.md

```diff
@@ -766,9 +769,33 @@ For class == "dcpm", the following parameters should be populated:
   for DAOS persistent storage mounted on the specified PMem device specified in
   `scm_list`.
 
+For class == "ram", `scm_list` is omitted and `scm_size` is specified
+instead.  In this case, the subsequent tiers describe the persistent storage
+used for both data and metadata.
+
+- `scm_size` specifies the amount of RAM dedicated to emulate SCM for holding
+  DAOS metadata.
+
 For class == "nvme", the following parameters should be populated:
 
 - `bdev_list` should be populated with NVMe PCI addresses.
+- `bdev_roles` optionally specifies a list of roles for this tier.
+
```

Comment:
These roles will be implicitly assigned if `bdev_roles` is not set, this is the default model and the role assignments should only be used if the implicit assignment does not cover the specific use case or intent that is required. Lets make that clear here.

Reply:
the exact heuristics used to implicitly assigned roles should be covered in the admin guide

Reply:
I've left the default/automatic assignment rules unspecified.  If we're pretty sure they are settled, it makes sense to define them, as they aren't very complicated.

Reply:
I just want it to be really really clear that the user does not need to explicitly specify the roles and that they should only be used in special circumstances, this will reduce the number of occurrences where we get strange behaviour because someone has someone has set unexpected role assignments.